resources/conf/log_config.toml

```diff
@@ -29,3 +29,6 @@ logLevel = "INFO"
 enable = false
 logfile = "/tmp/envoy.access.log" # This file will be created inside router container.
 format = "[%START_TIME%] '%REQ(:METHOD)% %REQ(X-ENVOY-ORIGINAL-PATH?:PATH)% %PROTOCOL%' %RESPONSE_CODE% %RESPONSE_FLAGS% %BYTES_RECEIVED% %BYTES_SENT% %DURATION% %RESP(X-ENVOY-UPSTREAM-SERVICE-TIME)% '%REQ(X-FORWARDED-FOR)%' '%REQ(USER-AGENT)%' '%REQ(X-REQUEST-ID)%' '%REQ(:AUTHORITY)%' '%UPSTREAM_HOST%'\n"
+
+[wireLogs]
+enable = false
```

Comment:
I prefer having wirelogs separately for payload and the headers. API Manager also follows the same approach. 
https://apim.docs.wso2.com/en/latest/observe/micro-integrator/classic-observability-logs/configuring-log4j2-properties/#wire-logs-and-header-logs
@pubudu538

Reply:
I guess in our case we buffer the body only when we need to deal with it. So +1 for your suggestion. I think we can have something like this.

[wireLogs]
enable = false
include = ["Body", "Headers","Trailers"]

Any other thoughts?

Reply:
I added the config with above format